Library/Formula/mesos.rb

```diff
@@ -15,22 +15,70 @@ class Mesos < Formula
   depends_on :apr => :build
   depends_on "subversion"
 
+  resource "boto" do
+    url "https://pypi.python.org/packages/source/b/boto/boto-2.36.0.tar.gz"
+    sha1 "f230ff9b041d3b43244086e38b7b6029450898be"
+  end
+
+  resource "protobuf" do
+    url "https://pypi.python.org/packages/source/p/protobuf/protobuf-2.6.1.tar.gz"
+    sha1 "3dff24d019729060eff569d7a718bdbb10db13a3"
+  end
+
+# build dependencies for protobuf
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha1 "d168e6d01f0900875c6ecebc97da72d0fda31129"
+  end
+
+  resource "python-dateutil" do
+    url "https://pypi.python.org/packages/source/p/python-dateutil/python-dateutil-2.4.0.tar.gz"
+    sha1 "159081a4c5b3602ab440a7db305f987c00ee8c6d"
+  end
+
+  resource "pytz" do
+    url "https://pypi.python.org/packages/source/p/pytz/pytz-2014.10.tar.bz2"
+    sha1 "74a1869c804dd422afbc49cb92206a0ca1529ddc"
+  end
+
+  resource "python-gflags" do
+    url "https://pypi.python.org/packages/source/p/python-gflags/python-gflags-2.0.tar.gz"
+    sha1 "1529a1102da2fc671f2a9a5e387ebabd1ceacbbf"
+  end
+
+  resource "google-apputils" do
+    url "https://pypi.python.org/packages/source/g/google-apputils/google-apputils-0.4.2.tar.gz"
+    sha1 "6f82069efd1a2cbc168dfb814d077df2fca4cff1"
+  end
 
   needs :cxx11
 
   def install
+    boto_path = libexec/"boto/lib/python2.7/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", boto_path
+    resource("boto").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"boto")
+    end
+    (lib/"python2.7/site-packages").mkpath
+    (lib/"python2.7/site-packages/homebrew-mesos-boto.pth").atomic_write boto_path + "\n"
```

Comment:
Also, can we just use `.write` here? I assume not but I'm maybe missing something.